### DISCUSSION TITLE
Duck Chess, too many clicks-per-move

### DISCUSSION THREAD

**Discussion by emanuelesaiu** [gbtami/pychess-variants#1474] [General]
When you play **Duck** on PyChess, **too many clicks/taps (or drags)** are needed for **each single move**. That really becomes a **serious practical problem** if you play with **fast** clock settings.
Instead, the chess dot com controls for Duck Chess are better: after setting the standard chess move, on chess dot com you need only one more click/tap in order to place the duck. That's much more reasonable.
I hope this little problem can be fixed in the next PyChess updates.
Thank you.

https://sites.google.com/site/emanuelesaiu/chess

**Comment by gbtami** [comment]
Filed as https://github.com/gbtami/pychess-variants/issues/1475

**Comment by gbtami** [comment]
@emanuelesaiu It is fixed in https://github.com/gbtami/pychess-variants/commit/649539f4120b4a955552af09a3d8e460ac2713f7

You can test it on https://pychess-variants-dev.onrender.com/
